src/containers/Application/ApplicationPageWrapper.tsx

```diff
@@ -61,16 +61,12 @@ const ApplicationPageWrapper: React.FC = () => {
   const { query, push, replace } = useRouter()
   const { mode, serialNumber, sectionCode, page } = query
 
-  const {
-    error,
-    loading,
-    application,
-    appStages,
-    templateSections,
-    isApplicationLoaded,
-  } = useLoadApplication({
-    serialNumber: serialNumber as string,
-  })
+  const { error, loading, application, templateSections, isApplicationLoaded } = useLoadApplication(
+    {
+      serialNumber: serialNumber as string,
+      networkFetch: true,
```

Comment:
I don't think you need `networkFetch: true` for this one. The idea was that we only need to force a network refresh when the application is likely to change in the back-end between fetches. Only when submitting the application is really an issue, the rest of the time the cached version should be fine. Maybe just comment out the line here (and in Overview?) so we can quickly switch it back on if/when we need to.

Reply:
Yes, good point. I was considering that we could have an status update here, but there shouldn't be one while the Applicant is working with the application.